packages/serverless-aws-s3/utils.js

```diff
@@ -38,6 +38,10 @@ const bucketCreation = async (s3, Bucket) => {
 
 const ensureBucket = async (s3, name, debug) => {
     try {
+        debug(`Checking bucket ${name} for uppercase check.`);
```

Comment:
![image](https://user-images.githubusercontent.com/5121148/96081910-94516400-0eba-11eb-845b-6943e6b06d52.png)

A couple of minor details.

1. Let's simplify `Checking bucket ${name} for uppercase check.`, and just make it `Validating S3 bucket name.`.
2. Message is good, just add a `.` at the end of the sentence, so it's unified with the rest of the messages.

Thanks!

Reply:
Modified